Python/Product/Profiling/Profiling.csproj

```diff
@@ -165,27 +142,16 @@
   </ItemGroup>
   <ItemGroup>
     <Content Include="proflaun.py">
-      <IncludeInVSIX>true</IncludeInVSIX>
```

Comment:
Does this file get included in the pylance.vsix now? I'll try it out later this afternoon.

Reply:
Yes, we need to check if all files that are not assemblies are included... I may have to update VSIX for Pylance manually.

Reply:
Let me try chaining. According to docs we may be able to keep multiple VSIX and instead nest them in Pylance vsix
https://docs.microsoft.com/en-us/visualstudio/extensibility/vsix-extension-schema-2-0-reference?view=vs-2019

**_Location** - This optional attribute specifies either the relative path within this VSIX to a nested VSIX package or a URL to the download location for the dependency. This attribute is used to help the user locate the prerequisite package._